extensions/frontend-tiles/src/GraphicsProvider/GraphicsProvider.ts

```diff
@@ -8,12 +8,12 @@ import { IModelApp, IModelConnection} from "@itwin/core-frontend";
 import { obtainGraphicRepresentationUrl} from "./GraphicRepresentationProvider";
 import { loggerCategory} from "../LoggerCategory";
 
-/** Arguments supplied  to [[obtainMeshExportTilesetUrl]].
+/** Arguments supplied  to [[obtainIModelTilesetUrl]].
  * @beta
  */
 export interface ObtainIModelTilesetUrlArgs {
   /** The iModel for which to obtain a tileset URl. */
-  iModel: IModelConnection;
+  iModel: IModelConnection | { iTwinId: string, iModelId: string, changesetId: string };
```

Comment:
`changesetId?: string` 
^^ optional right?

Reply:
Yeah it should be, fixed